engines/python/setup/djl_python/huggingface.py

```diff
@@ -72,6 +74,8 @@
     "QUESTION_ANS": AutoModelForQuestionAnswering,
 }
 
+OPENAI_CHAT_TEMPLATE = "{% for message in messages %}{% if message['role'] == 'user' %}{{ ' ' }}{% endif %}{{ message['content'] }}{% if not loop.last %}{{ '  ' }}{% endif %}{% endfor %}{{ eos_token }}"
```

Comment:
I know I suggested using a default template initially, but I think it will cause more issues than solve. I think we should remove this template, and use the tokenizer provided chat template. If no chat template is provided, we should throw an error to the user indicating they must use a model/tokenizer with a chat template

Reply:
OK